src/Services/Utils/utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Group of functions for transforming object or array of objects which keys have snake case,
+ * to camel case keys
+ */
+const fn = (key) => key.replace(/_([a-z])/g, (_, m) => m.toUpperCase());
+export const trasnform = (o) => Object.fromEntries(
+  Object.entries(o).map(([k, v]) => [fn(k), v]),
+);
```

Comment:
NIT: you need to keep either transform implementation or transformToCamelCase, as you are using only one